lxd/container_lxc.go

```diff
@@ -593,7 +595,8 @@ func containerLXCInstantiate(s *state.State, args db.ContainerArgs) *containerLX
 type containerLXC struct {
 	// Properties
 	architecture int
-	cType        db.ContainerType
+	instanceType instance.Type
```

Comment:
I don't see a reason not to call that just `type`

Reply:
@stgraber lower case "type" is a reserved word AFAIK.

Reply:
Especially since we're keeping `instance_type` to refer to the resource limit name thing, so having instanceType would just cause even more confusion

Reply:
https://golang.org/ref/spec#Keywords

Reply:
Oh, right, I wasn't expecting it to be reserved for struct members though.

How about dbType then? since it's technically the integer identifier that comes from there.

Reply:
@stgraber I'll double check but I did try "type" on the last iteration and it didnt work for some reason. If not then dbType looks good.

Reply:
Yeah, I just tried in a test file here and it definitely doesn't work even if only used inside a struct.